rpc/legacyrpc/server.go

```diff
@@ -238,6 +251,15 @@ func (s *Server) SetChainServer(chainClient *chain.RPCClient) {
 	s.handlerMu.Unlock()
 }
 
+// GetChainServer get the chain server client component needed to run a fully
+// functional decred wallet RPC server.
+func (s *Server) GetChainServer() *chain.RPCClient {
```

Comment:
I would name this `requireChainClient` and either use an ok bool or an error as the second return.